src/managers/GameManager.js

```diff
@@ -0,0 +1,62 @@
+export const getGames = () => {
```

Comment:
Same. All these API calls should be dried up. Eventually, you could have some kind of HTTPService or ApiService or something that wraps all of this and makes it dryer, but at first, you can just clean up each file with some variables.